### PR TITLE
added public functions get_members_by_source and get_sources_by_member

### PR DESCRIPTION
**Pull request by DartingMelody**
## General Checkups
- [x] Have you checked that there aren't other open pull requests for the same issue/update/change?
- [ ] If your code includes new features and not just bug fixes/copy editing: Did you include new tests?

### CODE REVIEW

**Comment by DartingMelody** [comment]
Hi @madprime , I have added the functions for endpoint  /api/public-data/sources-by-member/ and /api/public-data/members-by-source/ .  Please review. Thanks :)

File: ohapi/public.py [hunk 1]
```diff
@@ -168,3 +168,34 @@ def download(source=None, username=None, directory='.', max_size='128m',
         for value in executor.map(download_url_partial, results):
             if value:
                 logging.info(value)
+
+
+def get_data_members_by_source(base_url=BASE_URL_API):
```
Comment (madprime) [inline ohapi/public.py]: minor comment: the "data" in here doesn't attach to "members" in the API URL, it's a separate level, "public-data".

The function should be `get_members_by_source`, to be consistent with the API endpoint `members-by-source`.

File: ohapi/public.py [hunk 2]
```diff
@@ -168,3 +168,34 @@ def download(source=None, username=None, directory='.', max_size='128m',
...
+    Function returns which members have joined each activity.
+
+    :param base_url: It is URL: `https://www.openhumans.org/api/public-data`.
+    """
+    url = '{}members-by-source/'.format(base_url)
+    response = get_page(url)
+    return response
+
+
+def get_data_sources_by_member(base_url=BASE_URL_API, limit=LIMIT_DEFAULT):
```
Comment (madprime) [inline ohapi/public.py]: same naming update here – `sources-by-member` to match the endpoint URL

**Comment by DartingMelody** [comment]
Hi @madprime , I have made the changes. Please have a look. Thanks :)

**Comment by madprime** [comment]
Great, thanks! Merging now @DartingMelody ...

**Pull request merged**